observability/monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -119,8 +119,10 @@ endif::openshift-dedicated,openshift-rosa[]
 * xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#using-pod-topology-spread-constraints-for-monitoring_configuring-the-monitoring-stack[Using pod topology spread constraints for monitoring]
 * link:https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector[Kubernetes documentation about node selectors]
 
-include::modules/monitoring-moving-monitoring-components-to-different-nodes.adoc[leveloffset=+2]
-
+ifndef::openshift-dedicated,openshift-rosa[]
+include::modules/monitoring-moving-monitoring-components-to-different-nodes.adoc[leveloffset=+2,tags=**;CPM;!UWM]
```

Comment:
I am not sure that I understand the intent here. Is it:

If it's not OSD or ROSA (it's core OCP), then show CPM, and do not show UWM?

If so, I think the previews are showing what they should - OCP shows CPM, and ROSA/OSD shows UWM.

Just double checking, because I'm new to using tags.

Reply:
Yes, thank you for the questions! I really want to see what is confusing to peer review so that I can add some explanation comments and make it easier to review so thank you very much!

For OCP, it should show both CPM and UWM (they now show as two separate modules one after the another, this will not be the final way it will show to the user, I just need to separate the two procedures for now as the UWM modules wil go to its separate assembly, this is something it is long overdue for monitoring and is approved by all stakeholders, but I will think of a way to make that clearer for the reviewers so that it is not as confusing as I realise it is confusing for reviewers right now with the tagging).

For ROSA/OSD, it only shows UWM because CPM configuration is not supported there.